src/Paginator.php

```diff
@@ -29,6 +29,13 @@ class Paginator extends View
      */
     public $range = 4;
 
+    /**
+     * Set this if you want GET argument name to look beautifully.
+     *
+     * @var null|string
+     */
+    public $getArgName = null;
```

Comment:
Probably need to add something in docs about this property and getPageURL() method